web/packages/teleport/src/lib/global.d.ts

```diff
@@ -18,15 +18,15 @@
 
 import { rest } from 'msw';
 
-import type { SetupWorkerApi } from 'msw';
+import type { SetupWorker } from 'msw';
```

Comment:
It's [a breaking change from v1.0.0](https://github.com/mswjs/msw/releases/tag/v1.0.0).

> rename "SetupWorkerApi"/"SetupServerApi" types to "SetupWorker"/"SetupServer"

Though `SetupWorkerApi` is still there and TypeScript wasn't complaining about anything. 🤔